shapes/neurosciencegraph/datashapes/atlas/imagestack/schema.json

```diff
@@ -1,138 +1,143 @@
-            {
-              "path": "nsg:sliceInterval",
-              "name": "Slice interval",
-              "description": "Slice interval",
-              "node": "this:SliceIntervalShape",
-              "maxCount": 1
-            }
-          ]
-        }
-      ]
-    },
-    {
-      "@id": "this:SliceResolutionShape",
-      "@type": "sh:NodeShape",
-      "and": [
-        {
-          "node": "https://neuroshapes.org/commons/quantitativevalue/shapes/QuantitativeValueShape"
-        },
-        {
-          "property": [
-            {
-              "path": "nsg:widthResolution",
-              "name": "Width resolution",
-              "description": "Slice resolution in width",
-              "datatype": "xsd:float",
-              "minCount": 1,
-              "maxCount": 1
-            },
-            {
-              "path": "nsg:heightResolution",
-              "name": "Height resolution",
-              "description": "Slice height",
-              "datatype": "xsd:float",
-              "minCount": 1,
-              "maxCount": 1
-            }
-          ]
-        }
-      ]
-    },
-    {
-      "@id": "this:SliceIntervalShape",
-      "@type": "sh:NodeShape",
-      "and": [
-        {
-          "node": "https://neuroshapes.org/commons/quantitativevalue/shapes/QuantitativeValueShape"
-        },
-        {
-          "property": [
-            {
-              "path": "nsg:sliceIntervalValue",
-              "name": "Slice interval value",
-              "description": "Slice interval value",
-              "datatype": "xsd:float",
-              "minCount": 1,
-              "maxCount": 1
-            }
-          ]
-        }
-      ]
-    }
-  ]
-}
\ No newline at end of file
+  {
+    "@context": [
+      "https://incf.github.io/neuroshapes/contexts/schema.json",
+      {
+        "this": "https://neuroshapes.org/dash/imagestack/shapes/"
+      }
+    ],
+    "@id": "https://neuroshapes.org/dash/imagestack",
+    "@type": "nxv:Schema",
+    "import": [
+      "https://neuroshapes.org/commons/transformableobject",
+      "https://neuroshapes.org/commons/quantitativevalue"
+    ],
+    "prov:wasDerivedFrom": "https://github.com/INCF/neuroshapes/blob/v0.3.15/modules/atlas/src/main/resources/schemas/neurosciencegraph/atlas/imagestack/v1.0.0.json",
+    "shapes": [
+      {
+        "@id": "this:ImageStackShape",
+        "@type": "sh:NodeShape",
+        "label": "Image stack",
+        "nodeKind": "sh:BlankNodeOrIRI",
+        "targetClass": "nsg:ImageStack",
+        "and": [
+          {
+            "node": "https://neuroshapes.org/commons/transformableobject/shapes/TransformableObjectShape"
+          },
+          {
+            "property": [
+              {
+                "path": "nsg:imageModality",
+                "name": "Image modality",
+                "description": "Modality of the image stack",
+                "datatype": "xsd:string",
+                "maxCount": 1
+              },
```

Comment:
This property should probably be moved into the BrainImaging schema, since it refers to protocol information.